src/librustc/lint/builtin.rs

```diff
@@ -312,6 +312,12 @@ declare_lint! {
     "warn about documentation intra links resolution failure"
 }
 
+declare_lint! {
+    pub MISSING_DOC_ITEM_CODE_EXAMPLE,
```

Comment:
This may work better as `MISSING_DOC_CODE_EXAMPLES` - users don't need to care about the fact that docs go on items, and making it plural fits in better with other lints.

Reply:
I indeed prefer the new proposition. :)